RunningOrder/Sprint/SprintList.swift

```diff
@@ -51,8 +51,13 @@ struct SprintList: View {
             }
             .padding(.all, 8.0)
             .buttonStyle(PlainButtonStyle())
-        }.sheet(isPresented: $showNewSprintModal) {
-            NewSprintView(createdSprint: self.$sprints.appendedElement)
+        }
+        .listStyle(SidebarListStyle())
+        .sheet(isPresented: $showNewSprintModal) {
+            NewSprintView(createdSprint: $viewModel.sprints.appendedElement)
+        }
```

Comment:
why this change ?

Reply:
Because we design the SprintList like a Sidebar, I think it's more convenient to put the style here and not in the root view which use the list